### ISSUE TITLE
Reader: Audio Blocks Don't Appear

### ISSUE THREAD

**Issue by tmmbecker** [wordpress-mobile/WordPress-iOS#12578]
### Expected behavior

When loading a post with an audio block, I'd expect to see the audio files exactly as they appear in the Reader:

<img src="https://user-images.githubusercontent.com/22805865/65889916-09b1cb80-e370-11e9-8aa1-42e9f28c6168.png" width="200" alt="Screen Shot 2019-09-30 at 10 49 40">

### Actual behavior

No audio appears at all.

<img src="https://user-images.githubusercontent.com/22805865/65890177-76c56100-e370-11e9-94f2-76c6b4c12646.png" width="200" alt="Screen Shot 2019-09-30 at 10 52 34">

### Steps to reproduce the behavior

Create a post that includes the Audio block. Try to access it in the Reader. 

##### Tested on [device], iOS [version], WPiOS [version]

iPhone 7, iOS 13.1, WPiOS 13.2

Reported in 2370873-zen

**Comment by designsimply** [comment]
Tested and confirmed that the audio player does not show up on individual posts in the Reader in the app.

In the app, here is how an individual post looks in the Reader in light and dark mode (where you can see a bit of the outline of the player in a single line of pixels to the left and below where the player should be):

<img src="https://user-images.githubusercontent.com/1119271/65920727-16004d80-e39d-11e9-9fe5-498ccaeac7d8.PNG" width="270" alt="IMG_3114"> <img src="https://user-images.githubusercontent.com/1119271/65920735-1b5d9800-e39d-11e9-846e-bbbe0ccee668.PNG" width="270" alt="IMG_3113">
<sup>Tested with WPiOS 13.3.0.0 (TestFlight) on iPhone 6S iOS 13.1 beta 4 on `madefortesting568.wordpress.com`/`madefortesting568.com` (Premium Plan).</sup>

For comparison, here is how the post looks in the Reader on the mobile web:

<img src="https://user-images.githubusercontent.com/1119271/65920794-59f35280-e39d-11e9-93dc-963d595f193b.PNG" width="270" alt="IMG_3118">
<sup>Tested with Safari on iPhone 6S iOS 13.1 beta 4.</sup>

**Comment by designsimply** [comment]
I found this is still a problem while I was testing the Custom HTML block using the WPiOS 14.7.0.2 TestFlight beta release:

Steps to reproduce:

1. Go to https://raw.githubusercontent.com/cbracco/html5-test-page/master/index.html
1. Copy the content inside the `body` tags.
1. Create a new post in the app or on the web.
1. Add a paragraph block.
1. Add a Custom HTML block and paste the content copied in step 2.
1. Add another paragraph block.
1. Publish the post and view it in the Reader.

**Result:** audio does not appear.

Here are screenshots from Android and iOS to compare:

WPAndroid 14.7-rc-3 | WPiOS 14.7.0.2 TestFlight beta
---|---
<img src="https://user-images.githubusercontent.com/1119271/80546027-e1279980-8971-11ea-90b2-5cea0fd80b92.png" width="200" alt="Screenshot_20200428-124051">|<img src="https://user-images.githubusercontent.com/1119271/80545905-a1f94880-8971-11ea-92ea-01327f9442d9.PNG" width="200" alt="IMG_3936">

Sample audio HTML:

```
<audio controls="">audio</audio>
```

Interestingly, when I backgrounded the WPiOS app and checked it again, I saw this:

<img src="https://user-images.githubusercontent.com/1119271/80545968-bc332680-8971-11ea-9a05-5323b07e8521.PNG" width="200" alt="IMG_3938">
